packages/core/src/services/time-series.ts

```diff
@@ -104,8 +106,22 @@ export function formatTick(
 	}
 
 	const locale = timeScaleOptions.localeObject
-
-	return format(date, formatString, { locale })
+	const { code, optionsObject } = localeObject
+	if (interval === 'quarterly') {
```

Comment:
could you explain this piece pls? it seems like we're formatting date values using our `number` locale formatter